rst_files/jv.rst

```diff
@@ -677,9 +687,14 @@ Here's code to produce the 45 degree diagram
     scatter!(xs, ys, alpha=0.25, color=:green, lims=(0, plot_grid_max), ticks=ticks)
     plot!(xlabel=L"x_t", ylabel=L"x_{t+1}", guidefont=font(16))
 
+.. code-block:: julia 
+  :class: test 
 
-
-
+  @testset "More Solutions 1 Tests" begin
```

Comment:
Which ones? 

```
.. code-block:: julia
    using Random  
    Random.seed!(42)
    K = 50

    plot_grid_max, plot_grid_size = 1.2, 100
    plot_grid = range(0, stop = plot_grid_max, length = plot_grid_size)
    ticks = [0.25, 0.5, 0.75, 1.0]

    xs = []
    ys = []
    for x in plot_grid
        for i=1:K
            b = rand() < π_func(s(x)) ? 1 : 0
            U = rand(wp.F)
            y = h_func(x, b, U)
            push!(xs, x)
            push!(ys, y)
        end
    end

    plot(plot_grid, plot_grid, color=:black, linestyle=:dash, legend=:none)
    scatter!(xs, ys, alpha=0.25, color=:green, lims=(0, plot_grid_max), ticks=ticks)
    plot!(xlabel=L"x_t", ylabel=L"x_{t+1}", guidefont=font(16))

.. code-block:: julia 
  :class: test 

  @testset "More Solutions 1 Tests" begin
    @test [xs[4], ys[4]] ≈ [0.0, 0.44530420013435007]
    @test ticks == [0.25, 0.5, 0.75, 1.0]
    @test plot_grid[1] == 0.0 && plot_grid[end] == plot_grid_max && plot_grid_max == 1.2 && length(plot_grid) == plot_grid_size && plot_grid_size == 100
  end 
```

Reply:
Ah, the issue here is that the preceding code block is not running (probably), because the `using Random` is directly underneath the `.. code-block:: julia` declaration. I'll put some space in there and it should fix it.

Reply:
Done. Good catch.